### DISCUSSION TITLE
How to search for more tags?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#677] [Q&A]
The advanced search form implies it is possible to add several tags, categories, authors, etc.

How would I search for two tags?

Tried "tag 1", "tag " but it does not work even if both tags are used in a blog post.

**Comment by razonyang** [comment]
The **AND** operation for taxonomies(tags, categories...) is unsupported on current implementation. You can [request a new feature](https://github.com/razonyang/hugo-theme-bootstrap/issues/new?assignees=&labels=enhancement&template=feature_request.yml&title=Suggest+a+new+feature) if you need this feature, I'll find time to implement it.

BTW, **OR** operation is supported, such as `CSS | Alert` matches `CSS` or `Alert`.

**Comment by razonyang** [comment]
Implemented #679.
